streams/src/main/java/org/apache/kafka/streams/state/internals/AbstractRocksDBSegmentedBytesStore.java

```diff
@@ -225,6 +225,7 @@ public String name() {
         return name;
     }
 
+    @Deprecated
```

Comment:
There are a handful of these also, just passing the deprecation on to the callers.